src/FakeItEasy/FakeItEasy.csproj

```diff
@@ -95,5 +95,21 @@
   <ItemGroup>
     <None Include="$(OutputPath)/**/$(AssemblyName).pdb" Pack="true" PackagePath="lib" Visible="false" />
   </ItemGroup>
+
+  <ItemGroup>
+    <Service Include="{508349b6-6b84-4df5-91f0-309beebad82d}" />
```

Comment:
I figure this enables the template expansion. Is it worth commenting?

Reply:
I think it does. Visual Studio added it automatically. I don't think it's strictly necessary, but VS will probably keep adding it if we remove it. I'll add a comment.